src/annotator/test/sidebar-test.js

```diff
@@ -43,20 +43,24 @@ describe('Sidebar', () => {
   };
 
   /** Return the PortRPC instance for the first connected guest. */
-  const guestRPC = () => {
-    return fakePortRPCs[1];
+  const guestRPC = (index = 1) => {
+    return fakePortRPCs[index];
   };
 
-  const emitGuestEvent = (event, ...args) => {
+  const emitEventGuestN = (index = 1, event, ...args) => {
```

Comment:
Perhaps `emitNthGuestEvent` to match the naming pattern of the other functions?

Reply:
👍🏼